client/dashboard/sites/settings/site-actions.tsx

```diff
@@ -0,0 +1,88 @@
+import { useMutation } from '@tanstack/react-query';
+import {
+	__experimentalHeading as Heading,
+	Button,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { addQueryArgs } from '@wordpress/url';
+import { restoreSitePlanSoftwareMutation } from '../../app/queries';
+import { ActionList } from '../../components/action-list';
+import { DotcomFeatures } from '../../data/constants';
+import type { Site } from '../../data/types';
+
+const useRestorePlanSoftware = ( { slug, options }: Site ) => {
+	const mutation = useMutation( restoreSitePlanSoftwareMutation( slug ) );
+
+	if ( ! options.is_wpcom_atomic ) {
+		return null;
+	}
+
+	return {
+		title: __( 'Re-install plugins & themes' ),
+		description: __(
+			'If your website is missing plugins and themes that come with your plan you can re-install them here.'
+		),
+		actions: (
+			<Button
+				variant="secondary"
+				size="compact"
+				isBusy={ mutation.isPending }
+				onClick={ () => mutation.mutate() }
+			>
+				{ __( 'Restore' ) }
+			</Button>
+		),
+	};
+};
+
+const useCopySite = ( { capabilities, is_wpcom_staging_site, plan, slug }: Site ) => {
+	if (
+		! (
+			capabilities.manage_options &&
+			! is_wpcom_staging_site &&
```

Comment:
I don't really understand why we do this check. Shouldn't `plan?.features.active.includes( DotcomFeatures.COPY_SITE )` return false for this site (in the REST API I mean)

Reply:
No, they're the same. Unfortunately, the staging site seems to just set `is_wpcom_staging_site` to true 🥲

Reply:
Should we fix that in the REST API?

Reply:
Let me have a try!

Reply:
Proposed 183172-ghe-Automattic/wpcom